client/my-sites/sharing/connections/service-description.jsx

```diff
@@ -96,6 +96,17 @@ class SharingServiceDescription extends Component {
 					comment: 'Description for Eventbrite when no accounts are connected'
 				} );
 			},
+			instagram: function() {
+				if ( this.props.numberOfConnections > 0 ) {
+					return this.props.translate( 'Connected to your Instagram account.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Connected to your Eventbrite account.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).